index.html

```diff
@@ -166,55 +153,55 @@ <h2>Agenda</h2>
           <td>Welcome, admin, safety, CoC, goal of the day</td>
         </tr>
         <tr>
-          <td><pre>09:30 - 09:45<pre></td>
+          <td><pre>09:15 - 09:30</pre></td>
           <td>Introduction round, who are you, what do you want to get out of this</td>
         </tr>
         <tr>
-          <td><pre>09:45 - 10:30<pre></td>
+          <td><pre>09:45 - 10:30</pre></td>
```

Comment:
@RaoOfPhysics can you see it now? :)

Reply:
:( Forgot to follow-up with "Find and Replace" after I did "Copy and Paste". :/